lib/SerializerAppend2.js

```diff
@@ -16,7 +16,11 @@ const rename = promisify(fs.rename);
 const rimraf = promisify(_rimraf);
 const write = promisify(fs.writeFile);
 
-const nextPow2 = n => Math.pow(2, Math.ceil(Math.log(n) / Math.log(2)));
```

Comment:
The existing implementation had a bug where `nextPow2(n) === n` if n is a power of 2